message-analyzer-guide/installing-and-upgrading-message-analyzer.md

```diff
@@ -41,7 +41,7 @@ On this site, you will find details, new system requirements, brief installation
 > If you are installing Message Analyzer on a Windows 7 computer, you might experience a random reset of the TCP stack and a TCP connection loss. If this impacts an application, you may have to restart it. Otherwise, there is a possibility that you might have to restart your computer.
 
 > [!NOTE]
-> If you are installing Message Analyzer on a Windows 10 computer, the Windows Filtering Platform (WFP) message provider (Microsoft-PEF-WFP-MessageProvider) is installed as part of the operating system; therefore, the Message Analyzer installer (.msi) no longer installs this component. However, for all other supported down-level computers, such as Windows 8, the WFP message provider is installed with the Message Analyzer .msi application that you run from the download site. On Windows 10 computers, you can use this provider to conveniently capture traffic above the IP/Network Layer, which includes HTTP traffic, while removing lower-layer noise. Since the WFP provider is native to the Windows 10 operating system, you will not have to install any additional software to capture such traffic. To learn more about capturing this traffic on Windows 10 computers, see the WFP message provider PowerShell cmdlets on the TechNet [Windows and Windows Server Automation with PowerShell](https://technet.microsoft.com/library/dn249523.aspx) site.
+> If you are installing Message Analyzer on a Windows 10 computer, the Windows Filtering Platform (WFP) message provider (Microsoft-PEF-WFP-MessageProvider) is installed as part of the operating system; therefore, the Message Analyzer installer (.msi) no longer installs this component. However, for all other supported down-level computers, such as Windows 8, the WFP message provider is installed with the Message Analyzer .msi application that you run from the download site. On Windows 10 computers, you can use this provider to conveniently capture traffic above the IP/Network Layer, which includes HTTP traffic, while removing lower-layer noise. Since the WFP provider is native to the Windows 10 operating system, you will not have to install any additional software to capture such traffic. To learn more about capturing this traffic on Windows 10 computers, see the WFP message provider PowerShell cmdlets on the TechNet Windows and Windows Server Automation with PowerShell site.

```

Comment:
TechNet does not exist anymore, so I think we can just remove the last sentence:
```suggestion
> If you are installing Message Analyzer on a Windows 10 computer, the Windows Filtering Platform (WFP) message provider (Microsoft-PEF-WFP-MessageProvider) is installed as part of the operating system; therefore, the Message Analyzer installer (.msi) no longer installs this component. However, for all other supported down-level computers, such as Windows 8, the WFP message provider is installed with the Message Analyzer .msi application that you run from the download site. On Windows 10 computers, you can use this provider to conveniently capture traffic above the IP/Network Layer, which includes HTTP traffic, while removing lower-layer noise. Since the WFP provider is native to the Windows 10 operating system, you will not have to install any additional software to capture such traffic. 
```